src/qt/editor/processormimedata.cpp

```diff
@@ -40,13 +40,10 @@
 
 namespace inviwo {
 
-void ProcessorMimeData::ContextAwareDelete::operator()(Processor* p) const noexcept {
-    rendercontext::activateDefault();
-    delete p;
-}
-
-ProcessorMimeData::ProcessorMimeData(std::unique_ptr<Processor> processor)
-    : QMimeData{}, processor_{std::move(processor)} {
+ProcessorMimeData::ProcessorMimeData(std::shared_ptr<Processor> processor)
+    : QMimeData{}
+    , processor_{std::move(processor)}
+    , activateContext_{[]() { rendercontext::activateDefault(); }} {
```

Comment:
is `activateContext_` used?

Reply:
To activate the right context before deleting the processor.

Reply:
added a comment